stdlib/3/_thread.pyi

```diff
@@ -1,15 +1,22 @@
 # Stubs for _thread
 
-# NOTE: These are incomplete!
+from typing import Any, Callable, Dict, NoReturn, Tuple
 
-from typing import Any
+error = RuntimeError
 
 def _count() -> int: ...
 _dangling = ...  # type: Any
 
 class LockType:
-    def acquire(self) -> None: ...
+    def acquire(self, waitflag: int = ..., timeout: int = ...) -> None: ...
```

Comment:
According to the docs, `timeout` takes a floating point value and returns `bool`.